platform/qt/qt.cmake

```diff
@@ -201,6 +220,30 @@ if (NOT MBGL_QT_STATIC)
     )
 endif()
 
+include(CMakePackageConfigHelpers)
+set(CMAKECONFIG_INSTALL_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/qmapboxgl/)
+
+configure_package_config_file(
+	"platform/qt/QMapboxGLConfig.cmake.in"
+	"${CMAKE_CURRENT_BINARY_DIR}/QMapboxGLConfig.cmake"
```

Comment:
Perhaps putting all CMake package config files somewhere in more general, not under binary dir, is a bit more logical. "PACKAGE_DIR/cmake" or something. Then both libs/bin and include can be referenced from the same general place.

Reply:
As far as I remember, this is temporary destination. The install will put it into the correct location on your filesystem.

Reply:
No, the comment was was this line `INSTALL_DESTINATION ${CMAKECONFIG_INSTALL_DIR}`.

`CMAKECONFIG_INSTALL_DIR` is set to `${CMAKE_INSTALL_LIBDIR}/cmake/qmapboxgl/` but CMake cannot find the files when starting from top package dir. It starts to search like this: https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure

Reply:
Aha! I'm sorry. Now I understand the point of putting cmake config files under lib folder. This way the package can be installed to a system lib path and it can act both as system libs and CMake package. I reverted my change. Try that one:
https://github.com/kiibimees/maplibre-gl-native/commit/adeaaff713479ca1524c8adc5299e8d242a444f9